RELEASE-NOTE.md

```diff
@@ -4,6 +4,7 @@
  is the bug fix release.
 - Changes
   * Make default groovy maven project pom.xml work correctly.
+  * Add SSL JVM arguments.
```

Comment:
We actually use this arguments when make new worker process.

https://github.com/naver/ngrinder/blob/develop/ngrinder-core/src/main/java/net/grinder/engine/agent/AgentImplementationEx.java#L235

what about `Add SSL related JVM arguments to worker process command`?

Reply:
Change to `Drop SSLv2Hello protocol.`